src/h5web/vis-packs/core/heatmap/HeatmapVis.tsx

```diff
@@ -94,8 +94,8 @@ function HeatmapVis(props: Props) {
           flip: flipYAxis,
         }}
       >
-        <TooltipMesh {...tooltipFormatters} guides="both" />
         <PanZoomMesh />
+        <TooltipMesh {...tooltipFormatters} guides="both" />
```

Comment:
Should this be documented for `@h5web/lib` ?

Reply:
Went a bit too far with Storybook changes. Going to open a separate PR.